### PR TITLE
basic functionality, need clicks fixed

### PR DESCRIPTION
**Pull request by jywarren**
Fixes https://github.com/publiclab/leaflet-environmental-layers/issues/157

Need to get clicks working properly... debugging!

* [x] get the click to correctly open a popup
* [x] start storing the entire GeoJSON object in the found/matches storage so we can display properties;
    * with an object instead of an array, we should still be able to access found[0] and found[1]
    * modify 'closest()' to be able to handle Point features when scanning the entire set
    * note that it won't recognize `{ 0: 41, 1: -71 }` as the same as `[41, -71]` - instead it'll look for a `lat` and `lon` property of the object

### CODE REVIEW

**Comment by jywarren** [comment]
I'll be forking the upstream library with the extra changes, too, and will stop using a local copy very soon!

**Comment by jywarren** [comment]
OK, it's working here! https://jywarren.github.io/leaflet-environmental-layers/example/unearthing-pvd

Now, i pushed up my start to proper layer integration, instead of working in the example file.

**Comment by sagarpreet-chadha** [comment]
@jywarren , this layer is so cool!!! 

I am starting on integrating this layer into the LEL layer system, so that it can be used in `inline map syntax` in plots2. 
Will bump LEL version to npm also after testing this. Thanks!

File: src/unearthing.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        L.popup()
+          .setLatLng([point[0], point[1]])
+          .setContent(content)
+          .openOn(map);
+      },
+    });
+ 
+  });
+
+  return FracTracker_mobile ;
```
Comment (sagarpreet-chadha) [inline src/unearthing.js]: Hey Jeff, this looks incorrect?

**Comment by sagarpreet-chadha** [comment]
continued here: https://github.com/publiclab/leaflet-environmental-layers/pull/283

**Pull request closed without merging**